src/components/Editor/Editor.tsx

```diff
@@ -79,7 +79,27 @@ const Editor = (props: EditorProps) => {
 
   return (
     <LexicalComposer initialConfig={settings}>
-      <Box className='editor-container' bg={'text_area.bg_light'} _dark={{ bgColor: 'text_area.bg_dark' }}>
+      <Box
+        className='editor-container'
+        bg={'text_area.bg_light'}
+        _dark={{
+          bgColor: 'text_area.bg_dark',
+          borderColor: '#4A5568 !important',
+
+          '.editor-placeholder': {
+            color: 'rgba(255, 255, 255, 0.24)',
+          },
+
+          '&:hover': {
```

Comment:
Why don't use the `_hover` included in chakra? Wouldn't it be easier (less characters)?